maria-bucharest.md

```diff
@@ -0,0 +1,17 @@
+# Bucharest
+
+### Places to eat
+ - Jourmey Pub
+ - Van Gogh
+ - Eden
```

Comment:
I'm actually curious about the webinar for my colleagues who are learning GitHub :) and I make presentations for them